src/components/NetworkPanel/CyjsRenderer/CyjsRenderer.tsx

```diff
@@ -204,17 +204,18 @@ const CyjsRenderer = ({ network }: NetworkRendererProps): ReactElement => {
     console.log('#Time to  apply style: ', performance.now() - t1)
   }
 
-  // TODO: fix this function for the new apply mechanism
-  // const applyHoverStyle = (): void => {
-  //   if (cy != null) {
-  //     cy.nodes().removeClass('hovered')
-  //     cy.edges().removeClass('hovered')
-
-  //     if (hoveredElement != null) {
-  //       cy.getElementById(hoveredElement).addClass('hovered')
-  //     }
-  //   }
-  // }
+  const applyHoverUpdate = (): void => {
```

Comment:
This is currently implemented in the `CyjsRenderer` component.  Should this be moved to the `cyjs-utils` module?

Also do you have ideas on name should this function should have?